examples/parametric/plot_equivalent_width.py

```diff
@@ -145,15 +91,15 @@ def equivalent_width(grids, uv_index, index_window, blue_window, red_window):
         # --- define the parameters of the star formation and metal enrichment histories
         Z = grid.metallicity
         stellar_mass = 1e8
-        EqW = []
+        eqw = []
 
         # Compute each index for each metallicity in the grid.
         feature, blue, red = index_window[i], blue_window[i], red_window[i]
 
         for k in range(0, len(Z)):
-            EqW = get_ew(index, feature, blue, red, Z[k], stellar_mass, grid, EqW, 0)
+            eqw = measure_equivalent_width(index, feature, blue, red, Z[k], stellar_mass, grid, eqw, 0)

```

Comment:
See comment below. Here you should combine the eqws together for plotting, rather than in the function.